spec.html

```diff
@@ -13524,70 +13526,58 @@ <h1>
         1. Else if _hasParameterExpressions_ is *false*, then
           1. If *"arguments"* is an element of _functionNames_ or if *"arguments"* is an element of _lexicalNames_, then
             1. Set _argumentsObjectNeeded_ to *false*.
-        1. If _strict_ is *true* or if _hasParameterExpressions_ is *false*, then
-          1. NOTE: Only a single Environment Record is needed for the parameters, since calls to `eval` in strict mode code cannot create new bindings which are visible outside of the `eval`.
-          1. Let _env_ be the LexicalEnvironment of _calleeContext_.
-        1. Else,
-          1. NOTE: A separate Environment Record is needed to ensure that bindings created by direct eval calls in the formal parameter list are outside the environment where parameters are declared.
+        1. NOTE: If direct eval calls in the formal parameter list can create bindings which are visible outside of the `eval`, then a separate Environment Record is needed to ensure that such bindings do not affect the environment where parameters are declared.
```

Comment:
The previous note made it clear why strictness was relevant; this new wording requires the reader to know that strict calls to `eval` place bindings in a new inner scope. I would prefer to keep the previous notes, or to reword this so it does not require the reader to know that.

Reply:
I thought the following condition explained that, but will clarify by moving the note into the subsequent block.